### PR TITLE
fix: stats appear in tooltips now for most active contributors graph

### LINKED ISSUE #2076: Bug: Lists - Most Active contributors label shows no values
### Describe the bug

The label on the Most Active contributors chart isn't showing the numerical values for each bar.

image

### Steps to reproduce

1. Go to any list
2. Go to the Activity Tab
3. hover over chart to trigger label
4. label shows no values

### Browsers

_No response_

### Additional context (Is this in dev or production?)

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Fixes #2076

## Mobile & Desktop Screenshots/Recordings

**Before**

CleanShot 2023-11-08 at 17 41 52

**After**

CleanShot 2023-11-08 at 17 37 38

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: components/molecules/MostActiveContributorsCard/most-active-contributors-card.tsx [hunk 1]
```diff
@@ -217,16 +218,26 @@ function RowTooltip({
...
-                <div key={key} className="flex items-center gap-2 font-bold">
-                  <div className="w-3 h-3 rounded-sm" style={{ backgroundColor: value.color }}></div>
-                  <div className="font-light">{value.title}</div>
-                </div>
-              ))}
+              .map(([key, value]) => {
+                const { percentage, stat } = userStats[key as StatKeys];
+
+                return (
+                  <div key={key} className="grid grid-cols-[1fr,1fr] gap-2 font-light">
```
Comment (NsdHSO) [inline components/molecules/MostActiveContributorsCard/most-active-contributors-card.tsx:229]: `grid-cols-2` it is not equivalent with `grid-cols-[1fr,1fr]` ??

Comment (nickytonline) [reply components/molecules/MostActiveContributorsCard/most-active-contributors-card.tsx:229]: Good catch. Thanks @NsdHSO! I'm still new to Tailwind. 😅

Comment (nickytonline) [reply components/molecules/MostActiveContributorsCard/most-active-contributors-card.tsx]: Done!

File: components/molecules/MostActiveContributorsCard/most-active-contributors-card.tsx [hunk 2]
```diff
@@ -3,7 +3,6 @@ import { useGesture } from "@use-gesture/react";
 import Image from "next/image";
 import { ReactNode, useState } from "react";
 import * as Tooltip from "@radix-ui/react-tooltip";
-import clsx from "clsx";
```
Comment (nickytonline) [inline components/molecules/MostActiveContributorsCard/most-active-contributors-card.tsx:6]: This was used previously, but not required.

**Review by brandonroberts — APPROVED** [review]
🔢

**Review by OgDev-01 — APPROVED** [review]
LGTM

**Pull request merged**